src/PointwiseFunctions/AnalyticSolutions/RelativisticEuler/TovStar.cpp

```diff
@@ -459,6 +459,39 @@ void TovVariables<DataType, Region>::operator()(
   }
 }
 
+template <typename DataType, StarRegion Region>
+void TovVariables<DataType, Region>::operator()(
+    const gsl::not_null<tnsr::i<DataType, 3>*> dcoords_pressure,
+    [[maybe_unused]] const gsl::not_null<Cache*> cache,
+    Tags::DcoordsPressure<DataType> /*meta*/) const {
+  const auto& dr_pressure =
+      get(cache->get_var(*this, Tags::DrPressure<DataType>{}));
```

Comment:
move into the `else` branch so the cache evaluation only gets triggered when it's needed